documentation/SolutionsRegistryFormat.md

```diff
@@ -49,6 +49,12 @@ The `settingsHandlers` block is unique and one of the most important blocks in t
         "options": {
             "filename": "${{environment}.APPDATA}\\Texthelp Systems\\RWSettings10.xml"
         },
+        "supportedSettings": : {
+            "appsetting1": { ... appsetting1 metadata ... },
```

Comment:
This description isn't clear enough to tell what "appsetting1" actually is, relative to the name of a setting. It would help to give a concrete example.